cluster/images/hyperkube/README.md

```diff
@@ -6,6 +6,20 @@
 
 #### How to release by hand
 
+##### Quick steps
```

Comment:
Can we please make it clear that this is for a single container image for a specific architecture for development/CI scenarios?

Reply:
Thank you, @dims . Have refactored to bring out the development scenario aspect. PTAL.